samples/eShopLite/CatalogDb/Program.cs

```diff
@@ -1,9 +1,24 @@
 using CatalogDb;
+using Microsoft.EntityFrameworkCore;
+using Npgsql;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
-builder.AddNpgsqlDbContext<CatalogDbContext>("catalogdb");
+
+// Configure Npgsql.EntityFrameworkCore.PostgreSQL services
+if (builder.Configuration.GetConnectionString("catalogdb") is string { } connectionString)
```

Comment:
1. Do we have to have the `if` check? Can't we just pass the connection string on line 16?
2. If we need to keep this, the `string { }` is redundant.

```suggestion
if (builder.Configuration.GetConnectionString("catalogdb") is string connectionString)
```